src/antares/craft/exceptions/exceptions.py

```diff
@@ -12,6 +12,8 @@
 
 from typing import List
 
+from antares.craft.model.output import McType, ObjectType
```

Comment:
Generally speaking I think we prefer not having to import anything related to the project in this file as it's like an util file